pkgs/by-name/to/tokyonight-gtk-theme/package.nix

```
@@ -15,45 +19,107 @@ let
     "dark"
     "light"
   ];
-
+  sizeVariantList = [
+    "compact"
+    "standard"
+  ];
+  themeVariantList = [
+    "default"
+    "green"
+    "grey"
+    "orange"
+    "pink"
+    "purple"
+    "red"
+    "teal"
+    "yellow"
+    "all"
+  ];
+  tweakVersionList = [
```

```suggestion
  tweakVariantsList = [
```

for sake of simplicity, i would change this to also use `Variants`